src/render/Shader.cpp

```diff
@@ -152,7 +174,7 @@ unsigned int Shader::compileShader()
     glDeleteShader(fragmentShader);
     return 0;
   }
-
+ //std::cout << "compileShader shader 6" << std::endl;
```

Comment:
remove commented out debug info.... also consider a logging library to turn on/off DEBUG level info

Reply:
I removed them.